### PR TITLE
Changing to Oracle RAC doc update

### LINKED ISSUE #618: "Changing to Oracle RAC" page should be updated
The Oracle RAC database-related configuration docs are not been updated to instruct APIM 3.0.0 users and those are still being referred to the older configurations.

### PR DESCRIPTION
**Pull request by renuka-fernando**
### Changing to Oracle RAC
- Update doc
- fix #618

### CODE REVIEW

File: en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-oracle-rac.md [hunk 1]
```diff
@@ -1,206 +1,162 @@
...
+The following sections describe how to set up Oracle RAC to replace the default H2 database in your WSO2 product:
 
-!!! info
-To test products on Oracle RAC, rename `oracle_rac.sql` to `oracle.sql` before running `-Dsetup` .
+-   [Setting up the database and users](#setting-up-the-database-and-users)
+-   [Setting up the drivers](#setting-up-the-drivers)
+-   [Executing db scripts to create tables on Oracle RAC database](#executing-db-scripts-to-create-tables-on-oracle-rac-database)
 
+Oracle Real Application Clusters (RAC) is an option that facilitates clustering and high availability in Oracle database environments. In the Oracle RAC environment, some of the commands used in `oracle.sql` are considered inefficient. Therefore, the product has a separate SQL script, `oracle_rac.sql` for Oracle RAC. The Oracle RAC-friendly script is located in the `dbscripts` folder together with other `.sql` scripts.
```
Comment (npamudika) [inline en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-oracle-rac.md]: Shall we change the path as `<API-M_HOME>/dbscripts`?

File: en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-oracle-rac.md [hunk 2]
```diff
@@ -1,206 +1,162 @@
...
 
-### Setting up the JDBC driver
+### Setting up the drivers
 
 Copy the Oracle JDBC libraries (for example, the `<ORACLE_HOME>/jdbc/lib/ojdbc14.jar` file) to the `<PRODUCT_HOME>/repository/components/lib/` directory.
 
-!!! info
-Remove the old database driver from the `<PRODUCT_HOME>/repository/components/dropins` directory when you upgrade the database driver.
-
+1. Unzip the WSO2 API Manager pack. Let's call it `<API-M_HOME>`.
```
Comment (npamudika) [inline en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-oracle-rac.md]: Shall we change as "call it as `<API-M_HOME>`"

File: en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-oracle-rac.md (hunk 1 above)
Comment (renuka-fernando) [reply en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-oracle-rac.md]: fixed with https://github.com/wso2/docs-apim/pull/619/commits/2c49dea90e2c24ab57b4dbf7c8549ff78be98a05

File: en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-oracle-rac.md (hunk 2 above)
Comment (renuka-fernando) [reply en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-oracle-rac.md]: fixed with https://github.com/wso2/docs-apim/pull/619/commits/2c49dea90e2c24ab57b4dbf7c8549ff78be98a05

File: en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-oracle-rac.md [hunk 3]
```diff
@@ -1,206 +1,162 @@
...
+1.  Exit from the session by executing the `quit;` command.
 
-Copy the Oracle JDBC libraries (for example, the `<ORACLE_HOME>/jdbc/lib/ojdbc14.jar` file) to the `<PRODUCT_HOME>/repository/components/lib/` directory.
+### Setting up the drivers
 
-!!! info
-Remove the old database driver from the `<PRODUCT_HOME>/repository/components/dropins` directory when you upgrade the database driver.
+Copy the Oracle JDBC libraries (for example, the `<ORACLE_HOME>/jdbc/lib/ojdbc14.jar` file) to the `<API-M_HOME>/repository/components/lib/` directory.
 
+1. Unzip the WSO2 API Manager pack. Let's call it `<API-M_HOME>`.
```
Comment (npamudika) [inline en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-oracle-rac.md]: Shall we change like this?
```suggestion
1. Unzip the WSO2 API Manager pack. Let's call it as `<API-M_HOME>`.
```

**Pull request merged**